src/build.py

```diff
@@ -1243,12 +1243,12 @@ def run(sync_filter, build_filter, test_filter, options):
 
   if test_filter.Check('emwasm'):
     emscripten_wasm_out = CompileLLVMTortureBinaryen(
-        'Compile LLVM Torture (emscripten+wasm backend)',
+        'emwasm',
```

Comment:
I vote we go for consistency in the waterfall now, and try to come up with a better name than emwasm in a future CL.